res/mock_evm/src/lib.rs

```diff
@@ -65,25 +68,72 @@ impl MockEvmContract {
         assert_eq!(input.0.len(), 20);
     }
 
+    //
+    // OWNER CALL METHODS
+    //
+
+    #[allow(unused_variables)]
+    pub fn factory_update(&mut self, #[serializer(borsh)] bytes: Raw) {}
+
+    #[allow(unused_variables)]
+    pub fn factory_set_wnear_address(&mut self, #[serializer(borsh)] input: Raw20) {}
+
+    #[allow(unused_variables)]
+    pub fn deploy_upgrade(&mut self) {}
+
+    #[allow(unused_variables)]
+    pub fn resume_precompiles(&mut self, #[serializer(borsh)] paused_mask: u32) {}
+
+    #[allow(unused_variables)]
+    pub fn stage_upgrade(&mut self, #[serializer(borsh)] input: Raw) {}
+
+    pub fn state_migration(&mut self) {}
+
+    //
+    // AUTHORIZED CALL METHODS
+    //
+
+    #[allow(unused_variables)]
+    pub fn pause_precompiles(&mut self, #[serializer(borsh)] paused_mask: u32) {}
+
     //
     // SELF CALL METHODS
     //
 
     pub fn set_eth_connector_contract_data(&mut self, #[serializer(borsh)] _input: Raw) {}
 
-    pub fn set_paused_flags(&mut self, #[serializer(borsh)] _input: Raw) {}
+    #[allow(unused_variables)]
+    pub fn set_paused_flags(&mut self, #[serializer(borsh)] paused_mask: u32) {}
 
     //
     // CALLBACK HANDLER METHODS
     //
 
     #[result_serializer(borsh)]
-    pub fn factory_update_address_version(&mut self, #[serializer(borsh)] _input: Raw) -> u8 {
-        0
+    pub fn factory_update_address_version(&mut self, #[serializer(borsh)] _input: Raw) {
+    }
+
+    #[result_serializer(borsh)]
+    pub fn refund_on_error(&mut self, #[serializer(borsh)] _input: Raw) {
+    }
+
+    //
+    // TEST CALL METHODS
+    //
+
+    // TODO: verify return type
```

Comment:
Is still need to be verified? 😄

Reply:
No, this was removed, I think you were just commenting on a not refreshed github page. :)